tests/e2e/dynamic_provisioning.go

```diff
@@ -45,6 +46,7 @@ const OSC_REGION = "OSC_REGION"
 
 var _ = Describe("[bsu-csi-e2e] [single-az] Dynamic Provisioning", func() {
 	f := framework.NewDefaultFramework("bsu")
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
```

Comment:
Ok nice
https://linuxera.org/working-with-pod-security-standards/

Reply:
For the test, I intentionnaly did not look for a hardened security, we are free to do whatever we want